conftest.py

```diff
@@ -15,16 +15,15 @@
     pytest.register_assert_rewrite('borg.testsuite')
 
 
-import borg.cache
-from borg.logger import setup_logging
+import borg.cache                       # noqa: E402
+from borg.logger import setup_logging   # noqa: E402
 
 # Ensure that the loggers exist for all tests
 setup_logging()
 
-from borg.testsuite import has_lchflags, has_llfuse, has_pyfuse3
-from borg.testsuite import are_symlinks_supported, are_hardlinks_supported, is_utime_fully_supported
-from borg.testsuite.platform import fakeroot_detected, are_acls_working
-from borg import xattr
+from borg.testsuite import has_lchflags, has_llfuse, has_pyfuse3    # noqa: E402
+from borg.testsuite import are_symlinks_supported, are_hardlinks_supported, is_utime_fully_supported   # noqa: E402
+from borg.testsuite.platform import fakeroot_detected               # noqa: E402
```

Comment:
if you add a inline `# ...` comment, please have exactly 2 spaces between that and the code.

trying to align comments is a noble goal and looks pretty at first, but doesn't work in general (as you see in 25) and also leads to a lot of readjusting over time, when other lines get comments and then a lot of pre-existing comments need to be re-adjusted. so, it is easier and more regular to not try to align comments.